modules/source/mail/config/mail.xml

```diff
@@ -34,7 +38,7 @@
 			store-uri="${protocol}://${username:}:${password:}@${host}:${port}/${folder}"
```

Comment:
Why not put the properties inside the source xml file and expose the two Boolean false values as config properties of the source?